### ISSUE TITLE
Bootstrap 5

### ISSUE THREAD

**Issue by sdubois** [backdrop-contrib/views_bootstrap#71]
Is there any plan to support Bootstrap 5? It is currently the only officially supported version of Bootstrap https://endoflife.date/bootstrap

**Comment by stpaultim** [comment]
@bugfolder Do you have an idea of how much time it would take to get this to work with Bootstrap 5?

I'm giving some thought to this.

**Comment by robertgarrigos** [comment]
I'm starting to update this module to Bootstrap 5 in my own repo, branch 1.x-5.x (https://github.com/robertgarrigos/views_bootstrap/tree/1.x-5.x). For the moment only the carousel is working.

**Comment by bugfolder** [comment]
It seems likely that the same code would not support Bootstrap 3, 4, and 5; if as you develop, you find that to be the case, then there should either be a BS version switch in the settings, or consider a separate contrib module, `views_bootstrap5`. Since `bootstrap5_lite` is a separate theme from `bootstrap_lite`, the latter makes more sense to me.

**Comment by robertgarrigos** [comment]
Yep, I thought about having a separate contrib module. Provably it is the best.

**Comment by stpaultim** [comment]
Let's just assume that we're not going to make this module Bootstrap 5 compatible. But, we are encouraging the development of Views Bootstrap 5. I might take a stab at that sometime, but not yet.

**Comment by stpaultim** [comment]
I've updated the README to make this a bit more explicit and we can close this issue. 

@bugfolder I am going to issue a new release, I've patched a few minor bugs and added some outstanding feature request from myself that are pretty simple. 

* Nest accordions
* Keep more than one accordion open at the same time

Both are opt in features and should be backward compatible.

**Comment by bugfolder** [comment]
You're on a tear, @stpaultim ! Sounds good.
